### PR TITLE
The header

### LINKED ISSUE #40: Header
## Description
Once all the common elements have been fetched and the state has been populated as explained at [#37](https://github.com/frontity/frontity.org/issues/37), we have to create the `<Header />` component with its content, adapt it to the design (desktop and mobile), and add it to the `index.js`. Some things to consider:
* There is a dropdown submenu when hovering over the GitHub icon.
* The Twitter and GitHub icons change the color on hover. However, WordPress adds `svg`s as images, so we can't use the fill property. In the initial implementation we could just change the opacity on hover.
* There are going to be two different headers: `Common Header`, the one showing when a page is loaded, and `Fixed Header`, showing while scrolling.

You can see an example at current https://frontity.org.

### Common header
The content of this header is being fetched from [this endpoint](https://wp.frontity.org/wp-json/wp/v2/template-parts?slug=header), and you should be able to find it in Frontity `state`. We have to translate that content into a React component, matching Invision design. Apart from the Invision design, we can adapt the content of the header at the WordPress dashboard - Appearance - Template Parts.

* Desktop design:
We can find it at the page 1/8 of desktop design (homepage) at Invision.
Screenshot from 2020-03-16 09-51-59

* Mobile design (closed):
We can find it at the page 1/2 of mobile design at Invision.
Screenshot from 2020-03-16 09-55-19

* Mobile design (opened):
We can find it at the page 2/2 of mobile design at Invision.
Screenshot from 2020-03-17 12-49-22

### Fixed header
The content of this header is being fetched from [this endpoint](https://wp.frontity.org/wp-json/wp/v2/template-parts?slug=fixed-header), and you should be able to find it in Frontity `state`. We have to translate that content into a React component, matching Invision design. Apart from the Invision design, we can adapt the content of the header at the WordPress dashboard - Appearance - Template Parts.

* Desktop design:
We can find it at the page 5/8 (homepage) at Invision.
Screenshot from 2020-03-16 10-03-44

* Mobile design (closed):
There is not going to be fixed header in the mobile version. Instead we are going to use a scroll-to-top button #61

### PR DESCRIPTION
**Pull request by DAreRodz**
Closes #40

### CODE REVIEW

**Comment by frontibotito** [comment]
Deploy preview for _website_ ready!

Built with commit 69b2693dc7915a179c882c914a94526a90a553d5

https://frontity-org-npr0fg2w3.now.sh

File: packages/frontity-org-theme/src/components/header.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
+import { connect, css } from "frontity";
```
Comment (luisherranz) [inline packages/frontity-org-theme/src/components/header.tsx]: Testing.

**Comment by DAreRodz** [comment]
Thanks for the reviews, @michalczaplinski and @SantosGuillamot !

I think 1. and 2. can be fixed from Gutenberg, I'm gonna try that.

About 3, I'm pretty sure I discussed that with Mario and the solution he suggested was to add a floating button at the bottom of  the screen that scroll back to the top of the pageview. @SantosGuillamot, can you confirm that?

**Comment by luisherranz** [comment]
The blog link is also wrong but I guess that's also a Gutenberg problem.

Apart from that, it looks awesome, but the dropdown links are really hard to click:

ezgif com-video-to-gif

I don't know if that's me 🤷‍♂️

**Comment by DAreRodz** [comment]
Nah, you are right @luisherranz , the same is happening to Mario. I have to fix that.

**Comment by DAreRodz** [comment]
Code is ready, the link in the Frontity logo and the icons need to be changed from WordPress. I'll do that later.

**Comment by DAreRodz** [comment]
Oops! Yup, I forgot that. :upside_down_face:

**Comment by SantosGuillamot** [comment]
@DAreRodz As we talked I've updated the icons of the menu from WordPress. If it's easy let's try to align vertically to the middle the submenu icons.

**Pull request merged**